### PR TITLE
Uninstall command

### LINKED ISSUE #45: add uninstall command
should:
- accept tool name as arg (similar to install)
- doesn't need the all keyword found w/in install

should remove:
- the tool itself
- the symlink in the .local/.../tools directory

### PR DESCRIPTION
**Pull request by GreaterGoodest**
Fixes #45 by adding uninstall command. 

## Branching checklist
- [x] There is an issue associated with your PR (bug, feature, etc.. if not, create one)
- [x] Your PR description references the associated issue (i.e. fixes #123)
- [x] Code is in its own branch
- [x] Branch name is related to the PR contents
- [x] PR targets master

## Static analysis checks
- [x] All python files are formatted using black
- [x] All flake8 checks pass
- [x] All existing tests pass

## Documentation
- [x] Documentation about your PR is included in docs/ and the README, as appropriate

## Additional Tests
- [x] New code is unit tested
- [x] New tests pass

### CODE REVIEW

**Comment by GreaterGoodest** [comment]
Ignore this for now. It was previously passing the pipeline but no longer is.

**Comment by epi052** [comment]
> Ignore this for now. It was previously passing the pipeline but no longer is.

Was it the unmocked tool installs, specifically the ones that have `go` as a dependency?  

If that's the case, i just merged a fix for that.

**Comment by GreaterGoodest** [comment]
Tests complete, documentation updated, ready for review!

**Comment by epi052** [comment]
should be able to review tonight/tomorrow

File: pipeline/tools/subjack.yaml [hunk 1]
```diff
@@ -1,10 +1,14 @@
 installed: false
 dependencies: [go]
 go: &gotool !get_tool_path "{go[path]}"
-path: !join_path [!get_default "{gopath}", bin/subjack]
+path: &path !join_path [!get_default "{gopath}", bin/subjack]
 environ: {"GOPATH": !get_default "{gopath}"}
+home: &home !get_default "{home}"
```
Comment (epi052) [inline pipeline/tools/subjack.yaml:6]: at first glance, this isn't used anywhere.  is it necessary?

File: pipeline/tools/tko-subs.yaml [hunk 2]
```diff
@@ -1,9 +1,13 @@
 installed: false
 dependencies: [go]
 go: &gotool !get_tool_path "{go[path]}"
-path: !join_path [!get_default "{gopath}", bin/tko-subs]
+path: &path !join_path [!get_default "{gopath}", bin/tko-subs]
+home: &home !get_default "{home}"
```
Comment (epi052) [inline pipeline/tools/tko-subs.yaml:5]: necessary?

File: pipeline/tools/waybackurls.yaml [hunk 3]
```diff
@@ -1,8 +1,13 @@
 installed: false
 dependencies: [go]
 go: &gotool !get_tool_path "{go[path]}"
-path: !join_path [!get_default "{gopath}", bin/waybackurls]
+path: &path !join_path [!get_default "{gopath}", bin/waybackurls]
+home: &home !get_default "{home}"
```
Comment (epi052) [inline pipeline/tools/waybackurls.yaml:5]: necessary?

File: pipeline/tools/waybackurls.yaml [hunk 4]
```diff
@@ -1,8 +1,13 @@
...
+src: &src !join_path [*home, go, src, github.com, tomnomnom, waybackurls]
```
Comment (epi052) [inline pipeline/tools/waybackurls.yaml:6]: this looks like a holdover from the previous install method, can likely remove

File: pipeline/tools/webanalyze.yaml [hunk 5]
```diff
@@ -1,8 +1,12 @@
 installed: false
 dependencies: [go]
 go: &gotool !get_tool_path "{go[path]}"
-path: !join_path [!get_default "{gopath}", bin/webanalyze]
+path: &path !join_path [!get_default "{gopath}", bin/webanalyze]
+home: &home !get_default "{home}"
```
Comment (epi052) [inline pipeline/tools/webanalyze.yaml:5]: necessary?

**Comment by GreaterGoodest** [comment]
Requested changes integrated!

**Comment by epi052** [comment]
tyfys o7

**Pull request merged**